multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -126,6 +126,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("West Manga", "https://westmanga.info", "id", overrideVersionCode = 1),
         SingleLang("World Romance Translation", "https://wrt.my.id", "id", overrideVersionCode = 10),
         SingleLang("xCaliBR Scans", "https://xcalibrscans.com", "en", overrideVersionCode = 4),
+        SingleLang("Zahard", "https://zahard.xyz", "en", overrideVersionCode = -17),
```

Comment:
Can just leave the overrideVersionCode not set, there shouldn't be a problem about the version number jump.

```suggestion
        SingleLang("Zahard", "https://zahard.xyz", "en"),
```